cmd/lite.go

```diff
@@ -30,82 +42,135 @@ var liteCmd = &cobra.Command{
-		case err := <-autoLite.Errs():
-			switch errors.Cause(err).(type) {
-			case lite.ErrOldHeaderExpired:
-				// reobtain trust height and hash
-				return err
-			default:
-				// try with another full node
-				return err
-			}
-		}
-
-		return nil
+		return chain.StartLiteClient(filepath.Join(liteDir, chainID), updatePeriod)
 	},
 }
 
-// TODO: Figure out arguements for initializing a lite client with a root of trust
-var liteInitCmd = &cobra.Command{
-	Use:   "init [chain-id] [hash] [height] [trust-period]",
-	Short: "Create a new lite client for a configured chain, requires passing in a root of trust",
+var liteDeleteCmd = &cobra.Command{
+	Use:   "delete [chain-id]",
+	Short: "Delete an existing lite client for a configured chain, this will force new initialization during the next usage of the lite client.",
 	Args:  cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return nil
+		chainID := args[0]
+		chain, err := relayer.GetChain(chainID, config.c)
+		if err != nil {
+			return err
+		}
+
+		return chain.StopLiteClient()
 	},
 }
 
-// TODO: Figure out arguements for initializing a lite client with a root of trust
-var liteUpdateCmd = &cobra.Command{
-	Use:   "update [chain-id] [hash] [height] [trust-period]",
-	Short: "Update an existing lite client for a configured chain, requres passing in a root of trust",
-	Args:  cobra.ExactArgs(3),
+// Only checks headers in the trusted store - doesn't validate headers from the trusted node that don't yet exist in the lite client itself
+var liteGetHeaderCmd = &cobra.Command{
+	Use: "header [chain-id] [height]",
+	Short: "Get header from lite client. height at -1 returns first trusted header, 0 returns last trusted header and " +
+		"all others return the header at that height if stored in the lite client",
+	Args: cobra.RangeArgs(1, 2),
 	RunE: func(cmd *cobra.Command, args []string) error {
+		chainID := args[0]
+		chain, err := relayer.GetChain(chainID, config.c)
+		if err != nil {
+			return err
+		}
+		if chain.LiteClient != nil {
+			return fmt.Errorf("lite client is not running on this chain")
+		}
+		if len(args) == 1 {
+			header, err := chain.LiteClient.TrustedHeader(0, time.Now())
+			if err != nil {
+				return err
+			}
+			fmt.Print(header) // output
+		} else {
+			height, err := strconv.ParseInt(args[1], 10, 64) //convert to int64
+			if err != nil {
+				return err
+			}
+			if height == -1 {
+				height, err = chain.LiteClient.FirstTrustedHeight()
+				if err != nil {
+					return err
+				}
+			}
+			header, err := chain.LiteClient.TrustedHeader(height, time.Now())
+			if err != nil {
+				return err
+			}
+			fmt.Print(header) // output
+		}
 		return nil
 	},
 }
 
-var liteDeleteCmd = &cobra.Command{
-	Use:   "delete [chain-id]",
-	Short: "Delete an existing lite client for a configured chain, this will force new initialization during the next usage of the lite clien.",
-	Args:  cobra.ExactArgs(1),
+var liteGetValidatorsCmd = &cobra.Command{
```

Comment:
```suggestion
var liteValidatorsCmd = &cobra.Command{
```

https://golang.org/doc/effective_go.html#Getters